mkt/websites/indexers.py

```diff
@@ -138,3 +145,26 @@ def extract_document(cls, pk=None, obj=None):
             doc.update(cls.extract_field_analyzed_translations(obj, field))
 
         return doc
+
+    @classmethod
+    def strip_url(cls, url):
+        """
+        Remove all unwanted sections of the URL and return a string that will
+        be passed to Elasticsearch.
+
+        E.g. 'https://m.domain.com/topic/' will become 'domain/topic/', which
+        will further get tokenized by Elasticsearch into 'domain' and 'topic'.
+
```

Comment:
Mention somewhere that it can never be perfect (and it's ok) because we don't use a database of tlds